### PR TITLE
Updated upload_fixtures.py script

### PR DESCRIPTION
**Pull request by yonatann**
Now it uses specific queries from re:dash to populate the local datastore

### CODE REVIEW

**Comment by yonatann** [comment]
Added full text search to my forked repo.

**Comment by mushon** [comment]
:+1:

**Comment by yonatann** [comment]
I've modified the code according to your comments.
The only part that needs to be discussed is the date format.

File: open-budget-il/index.yaml [hunk 1]
```diff
@@ -114,6 +114,12 @@ indexes:
```
Comment (akariv) [inline open-budget-il/index.yaml]: why are all these changes in the indexes necessary?

File: open-budget-il/main.py [hunk 2]
```diff
@@ -191,13 +207,15 @@ def get(self,*args,**kw):
 
         data = None
         key = self.key(*args,**kw)
-        if key is not None:
+        if key is not None and self.request.get("no-cache") != "true":
             key = key + "//%s/%s/%s" % (self.first,self.limit,self.output_format)
             data = memcache.get(key)
+
         if data is not None:
+            logging.error("found key in cache")
```
Comment (akariv) [inline open-budget-il/main.py]: no error prints in every request please :)

File: open-budget-il/index.yaml (hunk 1 above)
Comment (yonatann) [inline open-budget-il/index.yaml]: I'm not sure - could this file be auto-generated?

Comment (akariv) [inline open-budget-il/index.yaml]: yes - if these were not intentional I'll just ignore them

**Pull request merged**